book/pipeline-service.md

```diff
@@ -1,29 +1,27 @@
 # Pipeline Service
 
-Pipeline Service provides Tekton APIs and services to kcp workspaces.
-When a workspace binds to the Pipeline Service APIExport, it has access to the following Tekton APIs:
+Pipeline Service provides a SaaS for pipelines. It leverages:
 
-- `Task`
-- `Pipeline`
-- `PipelineRun`
-- `Repository` (for [Pipelines as Code](https://pipelinesascode.com))
+- Kubernetes / OpenShift for the compute
+- Tekton Pipelines, Results and friends for the core of the service
+- OpenShift GitOps / Argo CD, Pipelines as Code for managing the infrastructure
 
-The following features are also provided:
+The following features are provided:
 
 - Signing and attestation of `TaskRuns` with [Tekton Chains](https://tekton.dev/docs/chains/)
-- Archiving of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
+- Archiving/Pruning of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
 - Integrations with GitHub, Gitlab, and Bitbucket with [Pipelines as Code](https://pipelinesascode.com) (PaC).
 
 ## Architecture
 
-Tekton controllers are deployed directly on the workload cluster.
-The kcp syncer for the workload cluster is configured to sync `Pipeline`, `Task`, and `PipelineRun` objects.
-When a `PipelineRun` is created on kcp, it is synced to the workload cluster whose controllers then create `TaskRun` objects and resulting `Pods`.
-The Chains and Results controllers then work with the synced `PipelineRun` and generated `TaskRun` objects to perform their respective actions.
+Pipeline Service is deployed onto the cluster alongside other RHTAP components. The following form the core architecture of Pipeline Service:
 
-Like upstream Tekton, Pipelines as Code (PaC) also runs controllers directly on workload clusters.
-Unlike other Tekton components, PaC also exposes a `Route`/`Ingress` that allows it to receive webhook events from source control repositories.
-To sent traffic from kcp to the workload cluster, Pipeline Service will deploy a "gateway" service that acts as a proxy to PaC on the workload cluster.
+a. OpenShift Pipelines: It enables the creation and management of Pipelines for user Applications.
+b. Pipelines as Code (PaC): An OpenShift Pipelines component that enables receiving webhook events from version control systems. User would install a Github App to configure this.
+c. Spray Proxy: Forwards requests to all clusters in the environment once an event is received on PaC. It is a reverse proxy that broadcasts to multiple backends.
+d. Tekton Results: Responsible for storing logs and pruning PipelineRuns. Utilises AWS RDS as the backend database.
+e. Tekton Chains: Provides PipelineRun attestation capabilities, ensuring that all PipelineRuns are verified and can be traced back to their source.
```

Comment:
I think github-flavored markdown didn't like the a, b, c enumeration. Can you check?

```suggestion
1. OpenShift Pipelines: It enables the creation and management of Pipelines for user Applications.
1. Pipelines as Code (PaC): An OpenShift Pipelines component that enables receiving webhook events from version control systems. User would install a Github App to configure this.
1. Spray Proxy: Forwards requests to all clusters in the environment once an event is received on PaC. It is a reverse proxy that broadcasts to multiple backends.
1. Tekton Results: Responsible for storing logs and pruning PipelineRuns. Utilises AWS RDS as the backend database.
1. Tekton Chains: Provides PipelineRun attestation capabilities, ensuring that all PipelineRuns are verified and can be traced back to their source.
```

Reply:
This is correct - numbered lists should use numbers, not letters:

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists

Reply:
Done.